src/views/catalog/customize/components/CustomizeSource/utils.ts

```diff
@@ -52,3 +60,27 @@ export const getPVCSource = (
 
   return dataVolumeSpec;
 };
+
+export const getTemplateStorageQuantity = (template: V1Template): string | undefined => {
+  const dataVolumeTemplates = getTemplateVirtualMachineObject(template)?.spec?.dataVolumeTemplates;
+
+  return dataVolumeTemplates?.[0]?.spec?.storage?.resources?.requests?.storage;
+};
+
+export const getHTTPHelperText = (template: V1Template, t: TFunction) => {
+  const importUrls = getTemplateImportURLs(template);
+
+  if (importUrls)
+    return t('Enter URL to download (for example: {{importUrls}}', {
```

Comment:
it's a list (same for registry), you should pick only one example out of the list ( first one? )

Reply:
Curious... The metadata attribute of template has the type `ObjectMetadata`.
```js
export declare type ObjectMetadata = {
    annotations?: {
        [key: string]: string;
    };
    ...
};
```
So every annotation has type `string` and not `string[]`. Do we have to change this type? Or maybe the annotation is a string formatted in a particular way?

Reply:
intersting, we will need to check that, did you find a template with this annotations ?